www/module.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
honestly, because webpack is weird, and even though this seems like it should do exactly the same thing as the webpack config, it fails without this file...